module-4/backend/week_four.md

```diff
@@ -5,15 +5,40 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's your favorite project management tool?
+  Waffle or an Either Pad with user choice anonymity to present innovative ideas or thoughts.
+  
 2. Why do we create staging environments?
+  Create a seperate hosted instance to represent a prod enviroment to test.
+  
 3. What are the characteristics of a good README (in your opinion)?
+  Intuitive simplistic instructions to getting a project started with follow up instructions for contribution and bugs.
+  
 4. What's one main improvement you're going to make to your code regarding accessibility issues?
+  Keyboard webpage accessibility
+  
 5. What are some basic security concerns to be aware of when building applications?
+  Authentication broken.
+  DB intrusions.
+  Cross Scripting.
+  
 6. Why is continuous integration helpful/important?
+  Constantly breaks down and rebuilds and application possibly catching intricate changes that will cause problems.
+  
 7. What are some continuous integration tools?
+  Travis CI 
+  Circle CI
 
 #### Review  
 
 8. What are some characteristics of "good" git workflow?
+  Good Communication
+  
 9. What are the four fundamental concepts of object oriented programming?
+  Abstraction
+  Encapsulation
+  Inheritance
+  Polymorphism
 10. What's a module in Ruby and what's the difference between a class and a module?
+  Module is a set of inheritable methods or executables that allow child classes to use if properly inherited
```

Comment:
we don't inherit modules into classes, we include them. classes inherit from each other. what's the purpose of a module? why do we use them?